pyproject.toml

```diff
@@ -3,7 +3,7 @@ line-length = 120
 target-version = 'py311'
 
 [tool.poetry]
-name = "redbox-copilot"
+name = "redbox-root"
```

Comment:
cant be called `redbox` otherwise we get:

```
The Poetry configuration is invalid:
  - Project name (redbox) is same as one of its dependencies
```